Scripts/main.ps1

```diff
@@ -11,20 +11,32 @@ Set-GitHubEnv -Name "EnvironmentID" -Value $EnvironmentID -Verbose
 Write-Output "$Task - Looking for monitoring Resource Group (RG):"
 $RGs = az group list -o json | ConvertFrom-JSON
 $RG = $RGs | Where-Object Name -match "$env:EnvironmentID-log"
-Set-GitHubEnv -Name "Monitoring_RGName" -Value $RG[0].name -Verbose
-Set-GitHubEnv -Name "Monitoring_RGID" -Value $RG[0].id -Verbose
+if($RG -ne $null){
+    Set-GitHubEnv -Name "Monitoring_RGName" -Value $RG[0].name -Verbose
+    Set-GitHubEnv -Name "Monitoring_RGID" -Value $RG[0].id -Verbose
+}else{
+    Write-Output "$Task - No Resource Group (RG) found named $env:EnvironmentID-log in subscription $env:SubscriptionName"
+}
 
 Write-Output "$Task - Looking for monitoring Log Analytics workspace (LAW):"
 $LAWs = az monitor log-analytics workspace list -o json | ConvertFrom-JSON
 $LAW = $LAWs | Where-Object Name -match "$env:EnvironmentID-log"
-Set-GitHubEnv -Name "Monitoring_LAWName" -Value $LAW[0].name -Verbose
-Set-GitHubEnv -Name "Monitoring_LAWID" -Value $LAW[0].id -Verbose
+if($LAW -ne $null){
```

Comment:
Put $null on the left side of the comparrison

Reply:
Fixed by removing comparison, now only checking if($LAW)